### PR TITLE
Patch 3

### PR DESCRIPTION
**Pull request by randomizerxd**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.py [hunk 1]
```diff
@@ -755,3 +759,5 @@ def main():
 
 if __name__ in ('__main__', '__builtin__', 'builtins'):
     main()
+
+register_module_line('urlscan.io', 'end', __line__())
```
Comment (JasBeilin) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.py:763]: Please remove this line.

Comment (JasBeilin) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.py]: Please remove this line.

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.py [hunk 2]
```diff
@@ -1,15 +1,20 @@
-import demistomock as demisto  # noqa: F401
-from CommonServerPython import *  # noqa: F401
+register_module_line('urlscan.io', 'start', __line__())
```
Comment (JasBeilin) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.py]: Please remove this line.

File: Packs/UrlScan/pack_metadata.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
     "name": "URLScan.io",
     "description": "urlscan.io Web Threat Intelligence",
     "support": "partner",
-    "currentVersion": "1.1.22",
+    "currentVersion": "1.2.0",
```
Comment (JasBeilin) [inline Packs/UrlScan/pack_metadata.json:5]: This seems like a revision to me, are you sure you want it as minor change?

**Comment by JasBeilin** [comment]
Please note my comment.
Also, please notice that opening a new PR instead of fixing the original, makes it hard for us to track it and help. You can Re-submit the same PR.
Also, since this is a new PR, you need to re-fill the contribution form so we can move foreward.
Thank you! :)

**Comment by randomizerxd** [comment]
Apologies for opening a new PR instead of modifying the original - I am not sure how to update an existing PR. I have filled out the contribution form.

File: Packs/UrlScan/pack_metadata.json (hunk 3 above)
Comment (randomizerxd) [reply Packs/UrlScan/pack_metadata.json]: Revision makes sense to me, I believe I have modified this to 1.1.23

**Comment by heipei** [comment]
Looks good, LGTM!

**Review by JasBeilin — APPROVED** [review]
Good Job!

**Pull request merged**